doc/api.yml

```diff
@@ -2451,6 +2451,40 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/school/files/share_link':
+    post:
+      tags:
+        - Files
+      summary: create a share link to access files or folder in owncloud
+      operationId: createShareLink
+      parameters:
+        - name: name
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: query
+          required: true
+          schema:
+            type: string
```

Comment:
add a description for both.
also add a pattern ( should path start with '/' ? should path contain '/', what about the name ?, can it contain spaces ?)